[2024-02-24T22:27:10.585-0600] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: drivers_to_bq.run_etl manual__2024-02-25T04:27:06.211999+00:00 [queued]>
[2024-02-24T22:27:10.587-0600] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: drivers_to_bq.run_etl manual__2024-02-25T04:27:06.211999+00:00 [queued]>
[2024-02-24T22:27:10.588-0600] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-02-24T22:27:10.594-0600] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_etl> on 2024-02-25 04:27:06.211999+00:00
[2024-02-24T22:27:10.598-0600] {standard_task_runner.py:60} INFO - Started process 11408 to run task
[2024-02-24T22:27:10.603-0600] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'drivers_to_bq', 'run_etl', 'manual__2024-02-25T04:27:06.211999+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/driversDag.py', '--cfg-path', '/var/folders/2c/plc265_s31s2djb7299kj1fh0000gn/T/tmpajdx83gy']
[2024-02-24T22:27:10.605-0600] {standard_task_runner.py:88} INFO - Job 7: Subtask run_etl
[2024-02-24T22:27:10.638-0600] {task_command.py:423} INFO - Running <TaskInstance: drivers_to_bq.run_etl manual__2024-02-25T04:27:06.211999+00:00 [running]> on host vamsees-macbook-pro.local
[2024-02-24T22:27:10.670-0600] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='drivers_to_bq' AIRFLOW_CTX_TASK_ID='run_etl' AIRFLOW_CTX_EXECUTION_DATE='2024-02-25T04:27:06.211999+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-25T04:27:06.211999+00:00'
[2024-02-24T22:27:10.671-0600] {driversDag.py:26} INFO - getting drivers data
[2024-02-24T22:28:46.172-0600] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-02-24T22:28:46.178-0600] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 11408. PIDs of all processes in the group: [11408]
[2024-02-24T22:28:46.178-0600] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 11408
